CMakeLists.txt

```diff
@@ -49,10 +49,10 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ${${QT_VERSIONED_PREFIX}_POSITION_INDEPENDEN
 if(UNIX)
     if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
         # using Clang
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Werror -Wno-pessimizing-move -Wno-mismatched-tags -Wno-unused-private-field -Wno-self-assign-overloaded")
```

Comment:
Could you please move this fix to a separate patch set. I also suggest introducing something like 'QTPROTOBUF_DEVELOPER_BUILD' or 'QTPROTOBUF_WARNINGS_AS_ERROR' cache variable to opt-in this functionality.

Reply:
oops, didn't mean to commit this. was a local workaround for the wall werror failing to build on my machine.